interfaces/acados_matlab/acados_ocp.m

```diff
@@ -110,8 +110,8 @@ function set(varargin)
 			elseif nargin==4
 				obj = varargin{1};
 				field = varargin{2};
-				stage = varargin{3};
-				value = varargin{4};
```

Comment:
Why? I would keep the value last, and put everything needed to route it before...

Reply:
In Matlab, it is common to have optional arguments.
The convention is to put the optional argument in the end.
This is also, how the `ocp_set` function is implemented.
Imagine, inside within `ocp_set`, `value` to be `prhs[5]` or `prhs[6]` depending if `stage` is passed.
Thus, it is more intuitive for a Matlab user to call the function like this, i.e. with stage as last and optional argument.